jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunExecutorActor.scala

```diff
@@ -207,14 +223,20 @@ class JobRunExecutorActor(
     case _ => stash()
   }
 
+  def receiveStartTimeout: Receive = {
+    case StartTimeout =>
+      log.info(s"Start timed out for JobRun ${jobRun.id} with deadline ${startingDeadline.get}")
```

Comment:
I would rather not deduce the the log... prefer "startingDeadline timeout expired..."  or something similar... "Start timed out..." isn't clear to me.   Along with jobRun.id and startingDeadline, it would be great to have createdAt.  Perhaps s"startingDeadline duration of ${startingDeadline.get} expired for JobRun ${jobRun.id} created at ${jobRun.createdAt}"